dijkstra.c

```diff
@@ -0,0 +1,125 @@
+#include "heap.h"
+#include "dijkstra.h"
+#include "characters.h"
+#include <stdio.h>
+
+
+//compares cost as apposed to sheaffers height map compare
+static int32_t cost_cmp(const void *key, const void *with) {
+  return ((struct character_s *)key)->cost - ((struct character_s * )with)-> cost;
```

Comment:
this make no sense? how to do you subtract functions? you need the tile too?. I dont know if just adding tile to parameter works or not since if it is a deferred call c doesn't have closures to keep tile

Reply:
fixed